### PR TITLE
Changes for fetching and saving instagram feeds

### PR DESCRIPTION
**Pull request by shamnadps**

### CODE REVIEW

File: backend/src/facebook-feed/facebook-feed.service.ts [hunk 1]
```diff
@@ -18,9 +18,7 @@ export class FacebookFeedService {
     }
 
     onModuleInit() {
-        setInterval(() => {
-            this.fetchAndSaveFacebookFeeds();
-        }, config.updateInterval);
+        this.fetchAndSaveFacebookFeeds();
```
Comment (walokra) [inline backend/src/facebook-feed/facebook-feed.service.ts]: It's a good practice to keep related changes together and make separate PRs to fixes.

Comment (shamnadps) [reply backend/src/facebook-feed/facebook-feed.service.ts]: Thank you :-) I have updated.
